lang/en/app.php

```diff
@@ -133,4 +133,7 @@
 
     'min_read' => '{count} min read',
     'word_count' => '{count} words',
+
+    'distance_format_km' => ':count km',
+    'distance_format_miles' => ':count miles',
```

Comment:
Instead of `count` I would prefer using `distance`; because ... it's not a count.
```suggestion
    'distance_format_km' => ':distance km',
    'distance_format_miles' => ':distance miles',
```
(this change requires another change in the php code)

Reply:
I'll change to distance.